src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,103 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
+
+    margin-bottom: unit(45px / @base-font-size-px, em);
+
+    .respond-to-min(1024px, {
+        margin-top: unit(60px / @base-font-size-px, em);
+    });
+
+    .image-container {
```

Comment:
This one probably shouldn't be aware of the mobile-carousel. I looked at some other CF patterns, how about a pattern similar to `content`:

```
.content
    .content_hero
    .content_bar
    .content_wrapper
        .content_sidebar
        .content_main
```

So I'd update media to use:
```
.media
    .media_wrapper
        .media_image-container
            .media_image
        .media_body
```

…or…

```
.media
    .media_image-container
        .media_image
    .media_body
```